.nsprc

```diff
@@ -2,6 +2,7 @@
   "exceptions": [
     "https://npmjs.com/advisories/39",
     "https://npmjs.com/advisories/48",
-    "https://npmjs.com/advisories/658"
+    "https://npmjs.com/advisories/658",
+    "https://npmjs.com/advisories/766"
```

Comment:
Weird that this shows up in the diff, given that it's already in `master`. But, whatevs...